files/image_config/caclmgrd/caclmgrd

```diff
@@ -61,12 +61,21 @@ class ControlPlaneAclManager(object):
 
     ACL_TABLE_TYPE_CTRLPLANE = "CTRLPLANE"
 
-    # To specify a port range, use iptables format: separate start and end
-    # ports with a colon, e.g., "1000:2000"
+    # To specify a port range instead of a single port, use iptables format:
+    # separate start and end ports with a colon, e.g., "1000:2000"
     ACL_SERVICES = {
-        "NTP": {"ip_protocols": ["udp"], "dst_ports": ["123"]},
-        "SNMP": {"ip_protocols": ["tcp", "udp"], "dst_ports": ["161"]},
-        "SSH": {"ip_protocols": ["tcp"], "dst_ports": ["22"]}
+        "NTP": {
+            "ip_protocols": ["udp"],
+            "dst_ports": ["123"]
+        },
+        "SNMP": {
+            "ip_protocols": ["tcp", "udp"],
+            "dst_ports": ["161"]
+        },
+        "SSH": {
+            "ip_protocols": ["tcp"],
+            "dst_ports": ["22"]
```

Comment:
what about telemetry, restapi services?

Reply:
what is the port used by sonic-mgmt-framework? @renukamanavalan , do you know?

Reply:
We will handle this in a separate PR.